configuration/amd64/mysql/add_v2xhub_user.sh

```diff
@@ -12,16 +19,17 @@ read -s -p "Please enter a password: "  PASS
 
 PASS_LENGTH=`echo $PASS | wc -c`
 
-if [ $PASS_LENGTH -ge 8 ] && echo $PASS | grep -q [a-z] && echo $PASS | grep -q [A-Z] && echo $PASS | grep -q [0-9] && echo $PASS | grep -q [\$\!\.\+_-\*@\#\^%\?~]; then
-    echo "Confirm password: "
-    read -s CONF_PASS
+if [ $PASS_LENGTH -ge 8 ] && echo $PASS | grep -q [a-z] && echo $PASS | grep -q [A-Z] && echo $PASS | grep -q [0-9] && ( echo $PASS | grep -q [\$\!\.\+_\*@\#\^%\?~] || echo $PASS | grep -q [-] ); then
+    echo
+    read -s -p "Confirm password: " CONF_PASS
     while [ $CONF_PASS != $PASS ]; do
-        echo "Passwords do not match. Please re-enter password: "
-        read -s CONF_PASS
+        echo
```

Comment:
Can this line be removed now

Reply:
the echo counteracts the -s and makes the read appear on a new line